diem-node/src/lib.rs

```diff
@@ -136,8 +136,8 @@ pub fn load_test_environment<R>(
     template.json_rpc.address = format!("0.0.0.0:{}", template.json_rpc.address.port())
         .parse()
         .unwrap();
+    template.api.address = template.json_rpc.address;
     template.json_rpc.stream_rpc.enabled = true;
-    template.api.enabled = true;
```

Comment:
Does this mean those only interested in dev api would configure `json rpc port` to set the dev api port in a validator config yaml? Or is this just some peculiarity for `load_test_environment `?

Reply:
The primary reason we need customize `address` here is because default `address` uses host `127.0.0.1`, and for local test node we want to change host to `0.0.0.0`, the port should be same.
API is enabled by default now, so there is no need to set `enabled` to true